backend/benefit/applications/tests/conftest.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+from applications.tests.factories import ApplicationFactory, UserFactory
+from django.contrib.auth.models import Permission
+from rest_framework.test import APIClient
+
+
+@pytest.fixture
+def api_client():
```

Comment:
Something I'm going to fix in kesäseteli is to change this as:
```
@pytest.fixture
def user():
    return UserFactory()


@pytest.fixture
def api_client(user):
    permissions = Permission.objects.all()
    user.user_permissions.set(permissions)
    client = APIClient()
    client.force_authenticate(user)
    return client
```
so that you can use the same user in the test if needed.

Reply:
ok, changed the code to match kesäseteli code. Because the user model is different I guess it's not worth it to make these fixtures shared between projects